filters/apiusagemonitoring/filter_test.go

```diff
@@ -0,0 +1,329 @@
+package apiusagemonitoring
+
+import (
+	"fmt"
+	"github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	"github.com/zalando/skipper/filters"
+	"github.com/zalando/skipper/filters/filtertest"
+	"github.com/zalando/skipper/metrics/metricstest"
+	"net/http"
+	"testing"
+)
+
+func init() {
```

Comment:
I do want to observe my debug log entries during test execution. The only way I found so far is to set the `Logger`'s level to `debug` in the `init` function. Open to suggestions for a better/cleaner way to achieve that.

Reply:
the init of the logging should go to the logging package if any, but in general we should avoid init() completely. If this only used during testing, then we can also put the init() to a apimonitor_test.go file, if we really have to. But I also suggest rather to just drop it. I often put temporary tracing or log config in the code while working on it, but I either don't commit those or drop them before opening the PR.

Reply:
I just see that this one is indeed a test file. I still would consider to drop it, because this one may enable debug logging for all the tests for everybody, working on other parts of the code.

Reply:
fixed in 16247fd92d4b46e79c293993ae82acc6c55c5334

Reply:
or alternatively, what do you think about putting it in the logging package somehow like this:

```
if os.Getenv("DEBUG") == true {
    log.Logger.SetLevel(logrus.DebugLevel)
}
```